dipy/workflows/reconst.py

```diff
@@ -793,16 +821,21 @@ def run(
             (from the command line) or [15, 4, 4] from a Python script to be
             converted to float and multiplied by 10**-4 . If None
             the fiber response function will be computed automatically.
-        extract_pam_values : bool, optional
-            Save or not to save pam volumes as single nifti files.
-        sh_order : int, optional
+        sphere_name : string, optional
+            Sphere name on which to reconstruct the fODFs.
+        relative_peak_threshold : float, optional
+            Only return peaks greater than ``relative_peak_threshold * m``
```

Comment:
Not sure how this will be rendered due to the asterisk.

Reply:
like this:

![image](https://github.com/user-attachments/assets/842c7047-e6d6-4b0e-8c6d-85410e48c958)